app/controllers/distributors_controller.rb

```diff
@@ -18,7 +18,8 @@ def create_params
     params.permit(
       :website_url,
       :image_url,
-      :contact_id
+      :contact_id,
+      :name
     )
   end
```

Comment:
Ahhh can you actually just have distributor return distributor.contact.name?

Reply:
Are you sure we want to tie these values together? I thought the contact could just be a random person's name (ie. Joe from Apex)